config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -1400,18 +1400,18 @@ periodics:
       - --gcp-node-image=gci
       - --gcp-nodes=1
       - --gcp-zone=us-west1-b
-      - --ginkgo-parallel=30
+      - --ginkgo-parallel=1
       - --image-family=cos-stable
       - --image-project=cos-cloud
       - --provider=gce
-      - --test_args=--ginkgo.focus=\[Feature:InPlacePodVerticalScaling\] --minStartupPods=2
+      - --test_args=--ginkgo.focus=\[Feature:InPlacePodVerticalScaling\] --minStartupPods=1
```

Comment:
Can you please add a comment here that we are not focusing on `[Serial]` because of reasons and give a link to a bug tracking this

Reply:
Sure, here goes..

Currently, in-place pod resize tests have two SIGDescribe jobs:
```golang
var _ = SIGDescribe("[Serial] Pod InPlace Resize Container (scheduler-focussed) [Feature:InPlacePodVerticalScaling]", func() {
        doPodResizeSchedulerTests()
})

var _ = SIGDescribe("Pod InPlace Resize Container [Feature:InPlacePodVerticalScaling]", func() {
        doPodResizeTests()
        doPodResizeResourceQuotaTests()
        doPodResizeErrorTests()
})
```

They have always been run in serial due to `--ginkgo-parallel=1` in the pull jobs and have a long history. This change makes the newly added CI jobs consistent with the long exercised pull jobs and runs all in-place pod resize tests in serial.

We do want to add a new CI job in beta (and maybe a new pull job as well) that runs all the in-place resize jobs that can be run in parallel with as much parallelism as possible within the limitations of the CI VM allocatable resources. This work item is tracked by issue https://github.com/kubernetes/kubernetes/issues/116431 and must be addressed by beta (ideally shortly after v1.27 release and code thaw)

Reply:
Sorry for not being clear. I understand the description update. In job definition it would be great to include `[Serial]` to the Focus or add a comment why it is not there.

I appreciate the update of a test name as well!

Reply:
```suggestion
      # this is not focusing on [Serial] because there is limited tests that can be run as not Serial and we have a bug tracking moving them to separate job: kubernetes/kubernetes#116431
      - --test_args=--ginkgo.focus=\[Feature:InPlacePodVerticalScaling\] --minStartupPods=1
```

Reply:
something like this ^^^

Reply:
🤦 
On the way :)